include/common.h

```diff
@@ -44,6 +44,17 @@ unsigned int memtest(unsigned int start, unsigned int end);
 #define COL8_008484     14
 #define COL8_848484     15
 #define MEM_MAP_ADDR (char *)((0x00078004)-(SYSSEG << 4))
+#define MAX_FREE_MEM_BLOCK 10
+#define LOW_MEM 0x10000 //current igd base address                        
+#define PAGING_MEMORY (64 * 1024 * 1024)  //currently we map 64M , i suppose currently is enough        
```

Comment:
I think PAGING_MEMORY should be detected via e820 area.

Reply:
Add max_paging_mem , this is store the max paging value base e820.

Reply:
as agreement, we increase the size to 512M